src/Tribe/Attendees_Table.php

```diff
@@ -1086,4 +1086,17 @@ public function search_box( $text, $input_id ) {
 
 		echo $search_box;
 	}
+
+	/**
```

Comment:
:no_entry_sign: Doc comment for parameter "$item" missing

<sub>hash: 6be9ca7e7077b9590c710b6b9cc251bd</sub>